pkg/manifests/csv.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: packageserver
   namespace: openshift-operator-lifecycle-manager
   labels:
-    olm.version: 0.19.0
+    olm.version: 0.0.0-6b8621226ef93b41e980e1c35f843e5c97f145a8
```

Comment:
I really don't like the behavior around this tag, but not using a constant tag breaks verify: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_operator-framework-olm/534/pull-ci-openshift-operator-framework-olm-master-verify/1688556770030522368

Reply:
@grokspawn yep! The linked dependent PR (https://github.com/openshift/operator-framework-olm/pull/533) fixes that and should merge first, after which I'll rebase this